pi5/config.go

```diff
@@ -35,39 +33,3 @@ func (cfg *Config) Validate(path string) ([]string, error) {
 	}
 	return []string{}, nil
 }
-
-// LinuxBoardConfig is a struct containing absolutely everything a genericlinux board might need
-// configured. It is a union of the configs for the customlinux boards and the genericlinux boards
-// with static pin definitions, because those components all use the same underlying code but have
-// different config types (e.g., only customlinux can change its pin definitions during
-// reconfiguration). The LinuxBoardConfig struct is a unification of the two of them. Whenever we
-// go through reconfiguration, we convert the provided config into a LinuxBoardConfig, and then
-// reconfigure based on it.
-type LinuxBoardConfig struct {
```

Comment:
This has been removed in favor of `Config`, up on line 11. The difference is just that `GpioMappings` doesn't exist, because it's a constant that can't be modified (because this isn't the customlinux implementation, where the pin definitions can change during reconfiguration).